applications/vanilla/views/post/comment.php

```diff
@@ -28,7 +28,7 @@
                     echo $this->Form->errors();
                     $this->fireEvent('BeforeBodyField');
 
-                    echo $this->Form->bodyBox('Body', ['Table' => 'Comment', 'FileUpload' => true]);
+                    echo $this->Form->bodyBox('Body', ['Table' => 'Comment', 'FileUpload' => true, 'placeholder' => t('Type your comment'), 'title' => t('Type your comment')]);
```

Comment:
Vanilla typically ends sentences like this with a period.

Reply:
@tburry do you want the period for the placeholders?

Reply:
I think placeholders are better without periods.